### ISSUE TITLE
Changing the Direction of the Refinement

### ISSUE THREAD

**Issue by MarkusAndres** [GfSE/SAF-Specification#73]
[devdoc/Operational Domain/Operational Capability Mapping Viewpoint.md](../blob/main/src/devdoc/Operational Domain/Operational Capability Mapping Viewpoint.md)

Would it make sense to change the direction of the SAF_StakeholderRequirementRefinement

**Comment by haarer** [comment]
I dont agree. Please see https://saf.gfse.org/userdoc/stereotypes.html#_19_0_3_6d8019d_1609148130261_947486_570

"SAF_StakeholderRequirementRefinement
Stereotype realizes multiple Concepts:

    Implementation of SAF Concept SHRrefiningCRN
    SHRrefiningCRN: Specifies the fact that a Stakeholder Concern is refined by Stakeholder Requirements.
    Implementation of SAF Concept SHRrefiningOSY
    SHRrefiningOSY: Specifies the fact that an Operational Story is refined by Stakeholder Requirements.
    Implementation of SAF Concept SHRrefiningOCY
    SHRrefiningOCY: Specifies the fact that an Operational Capability is refined by Stakeholder Requirements."

And please See also the diagram, especially the constraint.

https://saf.gfse.org/devdoc/Operational%20Domain/Stakeholder%20Requirement%20Definition%20Viewpoint.html#realization-of-exposed-concepts

The Stakeholder requirement depends on the capability, Story or concern, thus it must be the Client.
